packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -130,61 +130,55 @@ export const SelectTree = function SelectTree( {
 					}
 				) }
 			>
-				<label
-					htmlFor={ `${ props.id }-input` }
-					id={ `${ props.id }-label` }
-					className="woocommerce-experimental-select-control__label"
-				>
-					{ props.label }
-				</label>
-				{ props.multiple ? (
-					<ComboBox
-						comboBoxProps={ {
-							className:
-								'woocommerce-experimental-select-control__combo-box-wrapper',
-							role: 'combobox',
-							'aria-expanded': isOpen,
-							'aria-haspopup': 'tree',
-							'aria-labelledby': `${ props.id }-label`,
-							'aria-owns': `${ props.id }-menu`,
-						} }
-						inputProps={ inputProps }
-						suffix={ suffix }
-					>
-						<SelectedItems
-							isReadOnly={ isReadOnly }
-							items={ ( props.selected as Item[] ) || [] }
-							getItemLabel={ ( item ) => item?.label || '' }
-							getItemValue={ ( item ) => item?.value || '' }
-							onRemove={ ( item ) => {
-								if (
-									! Array.isArray( item ) &&
-									props.onRemove
-								) {
-									props.onRemove( item );
+				<BaseControl label={ props.label } id={ `${ props.id }-input` }>
```

Comment:
Was `${ props.id }-input` ever set as an `id` of anything before? I don't see that in the old code and am just wondering if it was a bug previously. This new code looks good. Just want to make sure I'm not missing anything.

Reply:
Yes. It is set on inputProps, line 77, and used both for the single and multiple select scenarios